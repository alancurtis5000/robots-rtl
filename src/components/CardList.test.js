import { render, cleanup  } from '@testing-library/react';
import CardList from './CardList';

describe("CardList",()=>{
    afterEach(cleanup);

    const mockRobots =[
      {
          id: 1,
          name:"John Snow",
          username:"JohnSnow",
          email:'johnSnow@gmail.com'
      }
    ]

    it('Should render',()=>{
        const {asFragment} = render(<CardList robots={mockRobots} />);
        expect(asFragment()).toMatchSnapshot();
    })
})