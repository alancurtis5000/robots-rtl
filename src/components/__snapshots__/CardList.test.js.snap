// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList Should render 1`] = `
<DocumentFragment>
  <div>
    <div
      class="tc grow bg-light-green br3 pa3 ma2 dib bw2 shadow-5"
    >
      <img
        alt="robots"
        src="https://robohash.org/1?size=200x200"
      />
      <div>
        <h2>
          John Snow
        </h2>
        <p>
          johnSnow@gmail.com
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
